### PR TITLE
chore: Update changelog for rc2

### PR DESCRIPTION
**Pull request by glnro**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -36,6 +36,22 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
+## [v8.0.0-rc2] - 2023-01-16
+
+* (gaia) Bump [ibc-go](https://github.com/cosmos/ibc-go) to [v3.4.0](https://github.com/cosmos/ibc-go/blob/v3.4.0/CHANGELOG.md) to fix a vulnerability in ICA. See [v3.4.0 CHANGELOG.md](https://github.com/cosmos/cosmos-sdk/blob/v0.45.9/CHANGELOG.md) and [v3.2.1 Release Notes](https://github.com/cosmos/ibc-go/releases/tag/v3.2.1) for details.
+* (gaia) Bump [cosmos-sdk](https://github.com/cosmos/cosmos-sdk) to [v0.45.11](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.45.11). See [CHANGELOG.md](https://github.com/cosmos/cosmos-sdk/blob/release/v0.45.x/CHANGELOG.md) for details.
+* (gaia) Bump [tendermint](https://github.com/tendermint/tendermint) to [0.34.24](https://github.com/tendermint/tendermint/tree/v0.34.24). See [CHANGELOG.md](https://github.com/tendermint/tendermint/blob/v0.34.24/CHANGELOG.md) for details.
+* (gaia) Bump [iquidity](https://github.com/Gravity-Devs/liquidity) to [v1.5.3](https://github.com/Gravity-Devs/liquidity/releases/tag/v1.5.3).
```
Comment (mpoke) [inline CHANGELOG.md]: nit: typo "iquidity"

**Review by mpoke — APPROVED** [review]
Nice work

File: CHANGELOG.md (hunk 1 above)
Comment (glnro) [reply CHANGELOG.md]: will update

**Comment by glnro** [comment]
@Mergifyio backport release/v8.0.x

**Pull request merged**